go/packages/dev-mode/main.go

```diff
@@ -91,8 +92,9 @@ func (e *Extension) ExternalExtension() {
 		for !(logQueue.Empty() && (force || receivedRuntimeDone)) {
 			logs, err := logQueue.Get(1)
 			if err != nil {
-				logger.Error(printPrefix, zap.Error(err))
-				return
+				lib.Error(err)
+				time.Sleep(time.Millisecond * 50)
+				continue
```

Comment:
have you encountered errors here or just you just adding it just in case?

Reply:
Yeah if we return here then it is possible we exit too early in the event of an initialization crash 👍